File: My-Blog-ServeOpen/appone/models/web_info.py
from django.db import models

class WebInfo(models.Model):
    web_name = models.CharField(max_length=16, verbose_name='网站名称')
    web_title = models.CharField(max_length=512, verbose_name='网站信息')
    notices = models.CharField(max_length=512, null=True, blank=True, verbose_name='公告')
    footer = models.CharField(max_length=256, verbose_name='页脚')
    background_image = models.CharField(max_length=256, null=True, blank=True, verbose_name='背景')
    avatar = models.CharField(max_length=256, verbose_name='头像')
    random_avatar = models.TextField(null=True, blank=True, verbose_name='随机头像')
    random_name = models.CharField(max_length=4096, null=True, blank=True, verbose_name='随机名称')
    random_cover = models.TextField(null=True, blank=True, verbose_name='随机封面')
    waifu_json = models.TextField(null=True, blank=True, verbose_name='看板娘消息')
    status = models.BooleanField(default=True, verbose_name='是否启用[0:否，1:是]')

    class Meta:
        db_table = 'web_info'
        verbose_name = '网站信息表'
        verbose_name_plural = verbose_name

    def __str__(self):
        return str(self.id) + '-' + str(self.web_name)

File: My-Blog-Go/models/web_info.go
package models

// WebInfo 网站信息表
type WebInfo struct {
	ID              uint    `gorm:"primaryKey;column:id" json:"id"`
	WebName         string  `gorm:"column:web_name;size:16" json:"webName"`
	WebTitle        string  `gorm:"column:web_title;size:512" json:"webTitle"`
	Notices         *string `gorm:"column:notices;size:512" json:"notices,omitempty"`
	Footer          string  `gorm:"column:footer;size:256" json:"footer"`
	BackgroundImage *string `gorm:"column:background_image;size:256" json:"backgroundImage,omitempty"`
	Avatar          string  `gorm:"column:avatar;size:256" json:"avatar"`
	RandomAvatar    *string `gorm:"column:random_avatar;type:text" json:"randomAvatar,omitempty"`
	RandomName      *string `gorm:"column:random_name;size:4096" json:"randomName,omitempty"`
	RandomCover     *string `gorm:"column:random_cover;type:text" json:"randomCover,omitempty"`
	WaifuJson       *string `gorm:"column:waifu_json;type:text" json:"waifuJson,omitempty"`
	Status          bool    `gorm:"column:status;default:true" json:"status"`
}

func (WebInfo) TableName() string {
	return "web_info"
}
